crates/contract-controller/Cargo.toml

```diff
@@ -28,7 +28,7 @@ overflow-checks = true
 
 [features]
 # for more explicit tests, cargo test --features=backtraces
-backtraces = ["cosmwasm-std/backtraces"]
+#backtraces = ["cosmwasm-std/backtraces"]
```

Comment:
Should this be commented out? If so should we delete it?

Reply:
Ahhh, yeah this was deprecated in the update cosmwasm-std version, I'll remove